### PR TITLE
Fix Time64Distro Packaging on GH Actions

### PR DESCRIPTION
**Pull request by liveans**
## Description

Fixes the packaging of Time64Distros

## Testing

CI

## Documentation

No

### CODE REVIEW

File: .github/workflows/package-linux.yml [hunk 1]
```diff
@@ -33,9 +33,9 @@ jobs:
           { config: "Release", os: "ubuntu-22.04", arch: "arm", tls: "openssl3" },
           { config: "Release", os: "ubuntu-22.04", arch: "arm64", tls: "openssl3" },
           { config: "Release", os: "ubuntu-22.04", arch: "x64", tls: "openssl3" },
-          { config: "Release", os: "ubuntu-24.04", arch: "arm", tls: "openssl3" },
```
Comment (nibanks) [inline .github/workflows/package-linux.yml:36]: Do we need to update the .azure pipelines too?

Comment (liveans) [reply .github/workflows/package-linux.yml:36]: Yes we should, I'll grab packages from GH actions first, verify the fix on docker image and will do changes on azure pipelines if it works.

File: scripts/package-distribution.ps1 [hunk 2]
```diff
@@ -6,6 +6,16 @@
 
 #>
 
+param (
+    [Parameter(Mandatory = $false)]
+    [switch]$Time64Distro
+)
+
+$Time64 = $false
```
Comment (ami-GS) [inline scripts/package-distribution.ps1:14]: I think $Time64Distro is already boolean. why do you need to add one more variable?

**Comment by ami-GS** [comment]
made PR against this forked repo
https://github.com/liveans/msquic/pull/1/files

File: .azure/OneBranch.PullRequest.yml [hunk 3]
```diff
@@ -201,6 +201,12 @@ extends:
       - build_linux
       jobs:
       - template: .azure/obtemplates/build-linux-packages.yml@self
```
Comment (nibanks) [inline .azure/OneBranch.PullRequest.yml:203]: Let's be explicit about the `kernel` parameter being passed in here, so the reader doesn't need to go look at the template to figure out the default. Same for Official yml.

**Pull request merged**